spec.emu

```diff
@@ -688,6 +688,30 @@ contributors: Google, Ecma International
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-temporal-ordinaltomonthcode" type="abstract operation">
+      <h1>
+        OrdinalToMonthCode (
+          _calendar_: a calendar type that is not *"iso8601"*,
+          _arithmeticYear_: an integer,
+          _ordinalMonth_: a positive integer,
+        ): a month code
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>
+          It returns the month code for the ordinal month mumber _ordinalMonth_ in _arithmeticYear_ of _calendar_. The given _month_ must exist in the given year.
+        </dd>
+      </dl>
+      <emu-alg>
+        1. Let _isoDate_ be ! CalendarIntegersToISO(_calendar_, _arithmeticYear_, _ordinalMonth_, 1).
+        1. Let _calendarFields_ be NonISOCalendarISOToDate(_calendar_, _isoDate_).
+        1. Assert: _calendarFields_.[[Year]] = _arithmeticYear_.
+        1. Assert: _calendarFields_.[[Month]] = _ordinalMonth_.
+        1. Assert: _calendarFields_.[[Day]] = 1.
+        1. Return _calendarFields_.[[MonthCode]].
```

Comment:
Note, this could also be something like

1. Let _fields_ be Calendar Fields Record { [[Year]]: _arithmeticYear_, [[Month]]: _ordinalMonth_, [[Day]]: 1 }.
2. Perform ! NonISOResolveFields(_calendar_, _fields_).
3. Return _fields_.[[MonthCode]].

I avoided doing that for now, since we still have a pending issue (#57) to move NonISOResolveFields into this spec and convert it into algorithm steps, and I thought having algorithm steps here would be preferred. Nonetheless, I think the solution with NonISOResolveFields would be clearer.